examples/counter.c

```diff
@@ -110,15 +110,13 @@ void step_clock(struct ssm_act *act) {
   switch (act->pc) {
   case 0:
     for (;;) {
-      ssm_later(ssm_to_sv(cont->clk), ssm_now() + (2 * SSM_SECOND),
-                ssm_marshal(1));
+      ssm_later(cont->clk, ssm_now() + (2 * SSM_SECOND), ssm_marshal(1));
```

Comment:
The PR changes `ssm_later` from accepting `ssm_sv_t` to a `ssm_value_t` and then turning that into a `sv` itself, it seems. Is there any reason why this is not uniform? E.g `ssm_sensitize` still requires me to `ssm_to_sv` myself.

Reply:
I am not sure which is the better choice, but I guess consistency is always good :)